community/maturity.md

```diff
@@ -0,0 +1,99 @@
+| **LC20** | Libraries that are mandatory dependencies of the project's code do not create more restrictions than the Apache License does.                                                     | **YES** All dependencies are listed.                                                                                                                                        |
+| **LC30** | The libraries mentioned in LC20 are available as Open Source software.                                                                                                            | **YES** All dependencies are listed are available as Open Source software                                                                                                   |
+| **LC40** | Committers are bound by an Individual Contributor Agreement (the "Apache iCLA") that defines which code they may commit and how they need to identify code that is not their own. | **YES** All committers have iCLAs.                                                                                                                                          |
+| **LC50** | The project clearly defines and documents the copyright ownership of everything that the project produces.                                                                        | **YES** And all source files are with APLv2 header, checked by `rat`.                                                                                                         |
+
+### Releases
+
+| **ID**   | **Description**                                                                                                                                                        | **Status**                                                                                                                                                                               |
+| -------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------- |------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **RE10** | Releases consist of source code, distributed using standard and open archive formats that are expected to stay readable in the long term.                              | **YES** Source release is distributed via [dist.apache.org](https://dist.apache.org/repos/dist/release/uniffle/) and linked from [download page](https://uniffle.apache.org/download/).  |
+| **RE20** | The project's PPMC (Project Management Committee, see CS10) approves each software release in order to make the release an act of the Foundation.                      | **YES** All releases have been voted at dev@opendal.apache.org and general@incubator.apache.org, and have at least 3 PPMC member's votes.                                                |
```

Comment:
`dev@opendal.apache.org`?

Reply:
I will correct this. This is a draft now.